MapboxNavigationTests/InstructionsBannerViewIntegrationTests.swift

```diff
@@ -101,12 +101,12 @@ class InstructionsBannerViewIntegrationTests: XCTestCase {
         let view = instructionsView()
         view.set(makeVisualInstruction(primaryInstruction: instructions, secondaryInstruction: nil))
 
+        //Slash should be present until an adjacent shield is downloaded
```

Comment:
Actually, with two shields next to each other, one exit and one destination shield, we should probably show the delimiter? However, exit types are yet to be added to MapboxDirections, IIRC? /cc @bsudekum

Reply:
Yeah the case you mentioned in this comment makes sense to me.